go/obscuronode/enclave/enclave.go

```diff
@@ -202,22 +203,22 @@ func (e *enclaveImpl) SubmitBlock(block types.Block) nodecommon.BlockSubmissionR
 		return nodecommon.BlockSubmissionResponse{IngestedBlock: false}
 	}
 
-	blockState := updateState(&block, e.blockResolver, e.txHandler, e.storage, e.storage)
+	blockState := updateState(&block, e.blockResolver, e.txHandler, e.storage, e.storage, e.nodeShortID)
 	if blockState == nil {
 		return e.noBlockStateBlockSubmissionResponse(&block)
 	}
 
 	// todo - A verifier node will not produce rollups, we can check the e.mining to get the node behaviour
 	hr, f := e.storage.FetchRollup(blockState.HeadRollup)
 	if !f {
-		panic("Should not happen")
+		panic("Failed to fetch rollup. Should not happen")
 	}
 	e.mempool.RemoveMempoolTxs(historicTxs(hr, e.storage))
 	r := e.produceRollup(&block, blockState)
 	// todo - should store proposal rollups in a different storage as they are ephemeral (round based)
 	e.storage.StoreRollup(r)
 
-	log.Log(fmt.Sprintf(">   Agg%d: Processed block: b_%d(%d)", obscurocommon.ShortAddress(e.node), obscurocommon.ShortHash(block.Hash()), block.NumberU64()))
+	log.Log(fmt.Sprintf(">   Agg%d: Processed block: b_%d(%d)", e.nodeShortID, obscurocommon.ShortHash(block.Hash()), block.NumberU64()))
```

Comment:
I tried this and the method calls ended up being longer :D Let me have another go.

Reply:
Haha fair enough, yeah it feels like it might be nice to have the formatting pattern in a single place but if it comes out convoluted it's no biggie. I've got like half a dozen temporary ones in there on my branch atm to debug and it's a PITA copying them everywhere so mostly self-interest there